### ISSUE TITLE
Why require stdscr?

### ISSUE THREAD

**Issue by pythonmcpi** [unicurses/unicurses#26]
The [readme](https://github.com/unicurses/unicurses#using-unicurses) states that you must initialize the screen using the following line:
```py
stdscr = initscr()
```

However, the following seems to work without issue:
```py
initscr() # This actually does work for me, even though the readme says it shouldn't
```
Other screen functions also seem to work as intended:
```py
doupdate() # no error
print(inch()) # prints out a number
print(getstr()) # gets a string and prints it
```

It also works fine if I import unicurses using the following:
```py
import unicurses as uc
```

So is the readme outdated, or am I missing something here?

(This is my first time using unicurses and I haven't tested to see if all the functions work. I'll update this issue if something breaks.)

**Comment by GiorgosXou** [comment]
> So is the readme outdated, or am I missing something here?

The readme is pretty outdated and I had no time to fix it

> However, the following seems to work without issue:

That's true, but it is generally a good practice to use the first way

> It also works fine if I import unicurses using the following:

That's true too, you can see an example of such use [here](https://github.com/GiorgosXou/TUIFIManager/blob/master/TUIFIManager/TUItilities.py#L4) too

> This is my first time using unicurses and I haven't tested to see if all the functions work. I'll update this issue if something breaks.)

I highly recommend you, to ckeck [this](https://github.com/unicurses/unicurses#getting-started) section of `readme.md` for that reason

**Comment by pythonmcpi** [comment]
> That's true, but it is generally a good practice to use the first way

I don't see how assigning `stdscr` is good practice when I (the user of unicurses) do not need it to call any functions.

**Comment by GiorgosXou** [comment]
which function requires it?

**Comment by GiorgosXou** [comment]
usually for every `wfunction(win)` there is an equivalent `function()`

**Comment by pythonmcpi** [comment]
> which function requires it?

- `getyx` (source: [`demos/test_background.py`](https://github.com/unicurses/unicurses/blob/master/demos/test_background.py#L5))
- `getmaxyx` (source: [`demos/test_background.py`](https://github.com/unicurses/unicurses/blob/master/demos/test_background.py#L17))
- `keypad` (source: [`demos/test_move_panels_advanced.py`](https://github.com/unicurses/unicurses/blob/master/demos/test_move_panels_advanced.py#L99))
- `set_borders_to` (source: [`demos/test_move_panels_advanced.py`](https://github.com/unicurses/unicurses/blob/master/demos/test_move_panels_advanced.py#L100))
- `wborder` (source: [`demos/test_panels_advanced.py`](https://github.com/unicurses/unicurses/blob/master/demos/test_panels_advanced.py#L61))

**Comment by GiorgosXou** [comment]
hahahaha true, sorry for that, it's too late for my brain right now

**Comment by GiorgosXou** [comment]
for `wborder` there's [`border`](https://github.com/unicurses/unicurses/blob/539b8647fc2bc59761b54156a3b51cdecc062b3f/unicurses/__init__.py#L1689) the rule of thumb is as i've said, for every `wfunction` there's probably a `function`

**Comment by pythonmcpi** [comment]
Yeah, I realized wborder after I posted the comment, so I edited it out

**Comment by pythonmcpi** [comment]
Also (this is unrelated to stdscr, but I don't want to open another issue just for this), `printw`/`wprintw` seems to be missing from both unicurses and curses (from the standard library). [PDCurses supports it](https://pdcurses.org/docs/MANUAL.html#printw) and so does [NCurses](https://linux.die.net/man/3/printw). Is there a reason why this was omitted? Or does it have another name?

**Comment by GiorgosXou** [comment]
I've never used it so I've never found out that existed *(I'll probably add it now that I know)*,  `addstr` `mvaddstr` and the `w...` equivalent functions, should do the same job when used  for example like `addstr(f'This is a  {variable}')`

**Comment by GiorgosXou** [comment]
*(feel free to make a pull request if you have the time to do so)*

**Comment by ttoommxx** [comment]
I wanted to add to this issue. It is not necessary to use the line
```
stdscr = initscr()
```
at all. In fact, the pointer `stdscr` is registered as a global variable by the library. You can try for yourself, by running:
```
import unicurses as uc
uc.initscr()
uc.waddstr(uc.stdscr, "hello world")
uc.getkey()
uc.endwin()
```

**Comment by hotlittlewhitedog** [comment]
I renamed "stdscr" as _stdscr, and it worked; until I saw in your doc that "stdscr" has never to be renamed !!! (never). So I changed the name back for safety and security... :D
